### PR TITLE
=ben interpreter benchmarks

### PR DESCRIPTION
**Pull request by 2m**
Results in the bench doc. It is interesting that bare OneBounded interpreter is faster than GraphInterpreter. However their performance evens out when they get wrapped into actors as is seen in the bench results (`FlowMapBenchmark` has the same results when using `GraphStage.Identity` or `Flow[Int].map(identity)`)

EDIT: link to [bench results doc](https://docs.google.com/spreadsheets/d/1Sdqx2qj_L6-gpiXerCZHQcmAWzgh9p-pN2N0lBi87Wk/edit?usp=sharing).

### CODE REVIEW

**Comment by drewhk** [comment]
Do you have the numbers somewhere?

File: akka-bench-jmh-dev/src/main/scala/akka/stream/InterpreterBenchmark.scala [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      sink.requestOne()
+      interpreter.execute(Int.MaxValue)
+    }
+    lock.acquire()
+  }
+
+  @Benchmark
+  @OperationsPerInvocation(100000)
+  def onebounded_interpreter_100k_elements() {
+    val lock = new Lock()
```
Comment (drewhk) [inline akka-bench-jmh-dev/src/main/scala/akka/stream/InterpreterBenchmark.scala]: why not a CountdownLatch?

Comment (2m) [inline akka-bench-jmh-dev/src/main/scala/akka/stream/InterpreterBenchmark.scala]: No real reason. Lock was used in the previous benchmarks.

**Comment by drewhk** [comment]
LGTM

**Pull request merged**